docs/guides/synthesize-unitary-operators.mdx

```diff
@@ -5,9 +5,9 @@ description: On the implementation of arbitrary unitary matrices on qubits
 
 # Synthesize unitary operations
 
-## What's required
+### What's required
```

Comment:
This is generally a bad practice in "semantic web", which is for accessibility + SEO. You're not supposed to jump from an h1 to h3.

Reply:
I know.  @javabster asked me to change it to an H3 because the H2 is so big.